### PR TITLE
fix #1720 - added semicolon to variable j, and stratified private funcs

### LINKED ISSUE #1720: read_oxy3_header prints a lot of imaginary numbers to the screen
**Describe the bug**
read_oxy3_header.p prints a lot of imaginary numbers to the screen, probably due to a trailing 'i' or 'j' in the function's code

**To Reproduce**
1. run the test function test_tutorial_nirs_multichannel20191023 (which is a currently failing test function for some reason).

**Expected behavior**
informative text feedback to the screen

**Screenshots**
a lot of ans = 0.0000 + 1.0000i

**Environment and versions (please complete the following information):**
 - Operating System: linux
 - MATLAB version: R2018b
 - FieldTrip version: master branch

**Additional context**
no hurry but perhaps @Horschig can have a look at some point.

### PR DESCRIPTION
**Pull request by Horschig**

### CODE REVIEW

**Comment by Horschig** [comment]
I am just seeing now that some changes trace back to the issue Helena had back then with wrong labelling of channels. I thought I committed those already, but apparently I was wrong. Apologie for that (to my excuse, these changes were made before I got a new laptop, probably forgot to push when I migrated - at least I didn't lose them).

**Review by schoffelen — CHANGES_REQUESTED** [review]
can you fix the conflict in ft_nirs_prepare_ODtransformation.m?

File: external/artinis/ft_nirs_prepare_ODtransformation.m [hunk 1]
```diff
@@ -185,13 +185,21 @@
 coefs = cell2mat(textscan(fid, '%f %f %f %f %f'));
 
 % extract all optode combinations that are relevant here
+<<<<<<< HEAD
```
Comment (schoffelen) [inline external/artinis/ft_nirs_prepare_ODtransformation.m:188]: one of the two alternatives is correct, and the <<<HEAD ===== lines need to be removed

**Review by schoffelen — CHANGES_REQUESTED** [review]
please fix the conflicts in ft_nirs_prepare_ODtransformation

File: external/artinis/ft_nirs_prepare_ODtransformation.m (hunk 1 above)
Comment (robertoostenveld) [reply external/artinis/ft_nirs_prepare_ODtransformation.m]: I just fixed it

Comment (robertoostenveld) [reply external/artinis/ft_nirs_prepare_ODtransformation.m]: oh... I might not have fixed it. I just selected one at random. So this still needs to be looked at

**Comment by schoffelen** [comment]
I guess the random guess was off. Looking at the file, the sens array is known as opto throughout. I changed it and will merge...

**Pull request merged**